### ISSUE TITLE
Not sending exceptions to App Insights?

### ISSUE THREAD

**Issue by bdschaap** [orellabac/ApplicationInsightsHelper#1]
Are there any problems automatically sending unhandled exceptions to App Insights?  

I see a message in my debug window showing the exception telemetry but never see it appear within App Insights.  The "Application started" custom event appears though.

**Comment by orellabac** [comment]
It should be a configuration issue. Are you sure that your telemeyty key is valid. You can create a small console app and try to send something to appinsights.

**Comment by bdschaap** [comment]
Yes thanks.  I'm able to see the "Application started" telemetry.  I can see the exception telemetry when I catch it and then call TrackException myself.  It just doesn't work for unhandled exceptions.

I just tried the article you referenced as well and it has the same effect.

**Comment by orellabac** [comment]
Did you tried adding  Application.ThreadException += Application_ThreadException; ? Some exceptions are only catched with this handler
